libbeat/docs/step-configure-credentials.asciidoc

```diff
@@ -0,0 +1,29 @@
+ifeval::["{beatname_lc}"!="winlogbeat"]
+. If you've secured Elasticsearch and Kibana, you need to pass credentials when
+you run the commands that set up and start {beatname_uc}. You can either pass
+credentials at the command line (as shown later in the getting started guide),
+or specify credentials in the config file. If you plan to run {beatname_uc} as a
+service, you must specify credentials in the config file.
+endif::[]
+ifeval::["{beatname_lc}"=="winlogbeat"]
+. If you've secured Elasticsearch and Kibana, you need to specify credentials
```

Comment:
Note: Since our instructions basically tell people to start Winlogbeat as a service, I figured it's work telling them directly up front to specify username/password.